ipn/conf.go

```diff
@@ -42,6 +42,10 @@ type ConfigVAlpha struct {
 	AutoUpdate      *AutoUpdatePrefs `json:",omitempty"`
 	ServeConfigTemp *ServeConfig     `json:",omitempty"` // TODO(bradfitz,maisem): make separate stable type for this
 
+	// StaticEndpoints are additional, user-defined endpoints that this node
+	// should advertize amongst its wireguard endpoints.
```

Comment:
one would think "advertize" would be the en_US variant of British-ending "advertise", but turns out even en_US uses "advertise" and the "advertize" is archaic at this point. TIL: https://books.google.com/ngrams/graph?content=advertize%2Cadvertise&year_start=1800&year_end=2000&corpus=en-2012&smoothing=3

Reply:
TIL 👍🏼 , also curious what happened around 1666 https://books.google.com/ngrams/graph?content=advertize%2Cadvertise&year_start=1500&year_end=2000&corpus=en-2012&smoothing=3 perhaps Shakespeare using lots of 'advertise'